src/clj/rems/api/schema.clj

```diff
@@ -44,7 +44,9 @@
           (s/optional-key :organization/last-modified) DateTime
           (s/optional-key :organization/owners) [User]
           (s/optional-key :organization/review-emails) [{:name LocalizedString
-                                                         :email s/Str}]}))
+                                                         :email s/Str}]
+          (s/optional-key :organization/enabled) s/Bool
+          (s/optional-key :organization/archived) s/Bool}))
```

Comment:
am I missing something or do existing organizations default to disabled and unarchived? wouldn't this need a migration?

Reply:
I don't think anyone is running new organization code yet so it should be ok? Also the disabling and archiving is not functional anywhere yet?

Reply:
Ok fair enough. But there should probably still be defaults for enabled & archived when creating an organization, right?

Reply:
Do you mean this part?

https://github.com/CSCfi/rems/blob/master/src/clj/rems/db/organizations.clj#L16

Reply:
Yeah!